bofire/data_models/domain/constraints.py

```diff
@@ -78,10 +92,10 @@ def is_fulfilled(self, experiments: pd.DataFrame, tol: float = 1e-6) -> pd.Serie
 
     def get(
         self,
-        includes: Union[Type, List[Type]] = Constraint,
-        excludes: Optional[Union[Type, List[Type]]] = None,
+        includes: Union[Type[C_], Sequence[Type[C_]]] = Constraint,
+        excludes: Optional[Union[Type[C__], List[Type[C__]]]] = None,
         exact: bool = False,
-    ) -> "Constraints":
+    ) -> "Constraints[C_]":
         """get constraints of the domain
 
         Args:
```

Comment:
Can you remove the types from the comments? They are wrong now and we don't need them anyway.

Reply:
When you say we don't need types in the docstrings, are they not used to generate the docs? For example [here](https://experimental-design.github.io/bofire/ref-constraints/#bofire.data_models.domain.constraints), the docs list the return type of the `get` method according to the docstring, even though this type differs from the return type in the function signature.

Reply:
If the doc-extractor is using the types in the docstrings we should fix the doc-extractor.

Reply:
Good point. Shall I leave the comments in for now, and then leave that to another PR? Or still take out the comments?

Reply:
Remove them. :)